### PR TITLE
Wars

### PR DESCRIPTION
**Pull request by Idhrendur**
Make script build latest changes
Make phony wars no longer phony

### CODE REVIEW

**Review by CaesarVincens — COMMENTED** [review]
Is there a particular reason the guarantees were removed? I know there were some changes discussed regarding great powers and their spheres.

File: HoI4Mod/Rome_Strikes_Back/history/countries/HUN - Hungary.txt [hunk 1]
```diff
@@ -115,11 +115,10 @@ add_opinion_modifier = { target = SOK modifier = positive_45 }
 add_opinion_modifier = { target = UTA modifier = positive_121 }
 
 
-
 add_ideas = {
 	HUN_bavarian_demostrations
 	disarmed_nation
-	isolation
```
Comment (CaesarVincens) [inline HoI4Mod/Rome_Strikes_Back/history/countries/HUN - Hungary.txt:122]: Hungary has isolation as part of its focus tree (focuses to switch out)

File: HoI4Mod/Rome_Strikes_Back/history/countries/RUS - Russia.txt [hunk 2]
```diff
@@ -165,7 +164,7 @@ add_ideas = {
 	RUS_ENG_alliance
 	RUS_ukrainian_separatism_1
 	volunteer_only
-	isolation
```
Comment (CaesarVincens) [inline HoI4Mod/Rome_Strikes_Back/history/countries/RUS - Russia.txt:168]: As with Hungary, switching out of isolation is part of their focus tree

**Comment by Idhrendur** [comment]
Yeah, the guarantees were removed as a result of removing the entire spherelings implementation.

**Pull request merged**